packages/flutter_tools/lib/src/commands/build.dart

```diff
@@ -61,11 +59,11 @@ class BuildCommand extends FlutterCommand {
   String get category => FlutterCommandCategory.project;
 
   @override
-  Future<FlutterCommandResult> runCommand() async => null;
+  Future<FlutterCommandResult> runCommand() async => FlutterCommandResult.success();
```

Comment:
IIUC, this would be run if someone tries doing `flutter build` without supplying a subcommand, which doesn't make sense. In that case, should this return `FlutterCommandResult.fail()`?